asteroid/data/sms_wsj_dataset.py

```diff
@@ -0,0 +1,233 @@
+        num_samples = example['num_samples']['observation']
+        if num_samples == self.seg_len or self.like_test:
+            rand_start = 0
+        else:
+            rand_start = np.random.randint(0, num_samples - self.seg_len)
+        if self.like_test:
+            stop = None
+        else:
+            stop = rand_start + self.seg_len
+        # Load mixture
+        x, _ = sf.read(audio_path[in_signal], start=rand_start,
+                       stop=stop, dtype='float32')
+        x = x.T
+
+        num_channels = self.target_dict['infos']['num_channels']
+        if self.single_channel:
+            if self.like_test:
+                ref_channel = 0
+            else:
+                ref_channel = np.random.randint(0, num_channels)
+            x = x[ref_channel]
+        seg_len = torch.as_tensor([x.shape[-1]])
+        # Load sources
+        source_arrays = []
+        for idx in range(self.n_src):
+            try:
+                s = 0
+                for t in target:
+                    if t == 'speech_source':
+                        start = 0
+                        stop_ = None
+                    else:
+                        start = rand_start
+                        stop_ = stop
+                    s_, _ = sf.read(audio_path[t][idx], start=start,
+                                    stop=stop_, dtype='float32')
+                    s += s_.T
+            except IndexError:
+                if self.single_channel:
+                    s = np.zeros((seg_len, ))
+                else:
+                    s = np.zeros((num_channels, seg_len))
+            source_arrays.append(s)
+
+        if target[0] == 'speech_source':
+            from sms_wsj.database.utils import extract_piece
+            offset = example['offset']
+            source_arrays = [extract_piece(s, offset_, num_samples)
+                             for s, offset_ in zip(source_arrays, offset)]
+            source_arrays = [s[rand_start:stop] for s in source_arrays]
+
+        sources = torch.from_numpy(np.stack(source_arrays, axis=0))
+        assert sources.shape[-1] == seg_len[0], (sources.shape, seg_len)
+        if self.single_channel and not target[0] == 'source':
+            sources = sources[:, ref_channel]
+
+        mixture = torch.from_numpy(x)
+
+        if self.normalize_audio:
+            m_std = mixture.std(-1, keepdim=True)
+            mixture = normalize_tensor_wav(mixture, eps=EPS, std=m_std)
+            sources = normalize_tensor_wav(sources, eps=EPS, std=m_std)
+        return mixture, sources
+
+    def get_infos(self):
+        """ Get dataset infos (for publishing models).
+
+        Returns:
+            dict, dataset infos with keys `dataset`, `task` and `target`.
+        """
+        infos = dict()
+        infos['dataset'] = self.dataset_name
+        infos['task_dataset'] = self.dset
+        infos['target'] = self.target
+        infos['license'] = [wsj0_license, wsj1_license, sms_wsj_license]
```

Comment:
Actually, it's licenses, sorry.
```suggestion
         infos['licenses'] = [wsj0_license, wsj1_license, sms_wsj_license]
```